doc/Library_Inventory.md

```diff
@@ -0,0 +1,484 @@
+| communication | Azure.Communication.CallAutomation | sdk/communication/Azure.Communication.CallAutomation |
+| communication | Azure.Communication.CallingServer | sdk/communication/Azure.Communication.CallingServer |
+| communication | Azure.Communication.Chat | sdk/communication/Azure.Communication.Chat |
+| communication | Azure.Communication.Email | sdk/communication/Azure.Communication.Email |
+| communication | Azure.Communication.Identity | sdk/communication/Azure.Communication.Identity |
+| communication | Azure.Communication.PhoneNumbers | sdk/communication/Azure.Communication.PhoneNumbers |
+| communication | Azure.Communication.Rooms | sdk/communication/Azure.Communication.Rooms |
+| communication | Azure.Communication.ShortCodes | sdk/communication/Azure.Communication.ShortCodes |
+| communication | Azure.Communication.Sms | sdk/communication/Azure.Communication.Sms |
+| confidentialledger | Azure.Security.ConfidentialLedger | sdk/confidentialledger/Azure.Security.ConfidentialLedger |
+| containerregistry | Azure.Containers.ContainerRegistry | sdk/containerregistry/Azure.Containers.ContainerRegistry |
+| core | Azure.Core.Expressions.DataFactory | sdk/core/Azure.Core.Expressions.DataFactory |
+| core | Azure.Core.TestFramework | sdk/core/Azure.Core.TestFramework |
+| deviceupdate | Azure.IoT.DeviceUpdate | sdk/deviceupdate/Azure.IoT.DeviceUpdate |
+| digitaltwins | Azure.DigitalTwins.Core | sdk/digitaltwins/Azure.DigitalTwins.Core |
+| eventgrid | Azure.Messaging.EventGrid | sdk/eventgrid/Azure.Messaging.EventGrid |
+| formrecognizer | Azure.AI.FormRecognizer | sdk/formrecognizer/Azure.AI.FormRecognizer |
+| hdinsight | Microsoft.Azure.HDInsight.Job | sdk/hdinsight/Microsoft.Azure.HDInsight.Job |
+| iot | Azure.IoT.Hub.Service | sdk/iot/Azure.IoT.Hub.Service |
+| maps | Azure.Maps.Common | sdk/maps/Azure.Maps.Common |
+| maps | Azure.Maps.Geolocation | sdk/maps/Azure.Maps.Geolocation |
+| maps | Azure.Maps.Rendering | sdk/maps/Azure.Maps.Rendering |
+| maps | Azure.Maps.Routing | sdk/maps/Azure.Maps.Routing |
+| maps | Azure.Maps.Search | sdk/maps/Azure.Maps.Search |
+| maps | Azure.Maps.TimeZones | sdk/maps/Azure.Maps.TimeZones |
+| maps | Azure.Maps.Weather | sdk/maps/Azure.Maps.Weather |
+| metricsadvisor | Azure.AI.MetricsAdvisor | sdk/metricsadvisor/Azure.AI.MetricsAdvisor |
+| mixedreality | Azure.MixedReality.Authentication | sdk/mixedreality/Azure.MixedReality.Authentication |
+| monitor | Azure.Monitor.Ingestion | sdk/monitor/Azure.Monitor.Ingestion |
+| monitor | Azure.Monitor.OpenTelemetry.Exporter | sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter |
+| monitor | Azure.Monitor.OpenTelemetry.LiveMetrics | sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics |
+| monitor | Azure.Monitor.Query | sdk/monitor/Azure.Monitor.Query |
+| objectanchors | Azure.MixedReality.ObjectAnchors.Conversion | sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion |
+| openai | Azure.AI.OpenAI | sdk/openai/Azure.AI.OpenAI |
```

Comment:
This uses tsp and the new generator via special handling. We can hardcode this in the tsp new generator table.